### ISSUE TITLE
Goerli ETH Market

### ISSUE THREAD

**Issue by nambrot** [hyperlane-xyz/hyperlane-monorepo#1810]

**Comment by nambrot** [comment]
aroralanuk

**Comment by anettrolikova** [comment]
Relevant discussion https://www.reddit.com/r/ethereum/comments/ywv6o7/marketplace_for_goerli_eth/

**Comment by anettrolikova** [comment]
Goerli ETH marketplace https://bigoerlifaucet.com/

**Comment by anettrolikova** [comment]
Arbitrum  
https://defillama.com/chain/Arbitrum -> TVL 
https://defiprime.com/arbitrum#analytics -> dapps

**Comment by anettrolikova** [comment]
"what we are building is a marketplace for goerli eth, that could be accessed from either mainnet eth or arbitrum, and allow swaps of eth for goerli eth. we have most of the contract work already done, and deployment needs to happen. " @Skunkchain

**Comment by anettrolikova** [comment]
buy goerli with eth 
https://mirror.xyz/0xf3Ae96eF494927cdEdBFe4cEED5671556d4437a6/tiNiX8IC_PP4cp-CdwpiiEQTDluTKQxYyQ6zUlZfRfU

**Comment by anettrolikova** [comment]
<img width="518" alt="image" src="https://user-images.githubusercontent.com/44020788/218565156-ec9f89f9-56aa-4185-b3e1-a77487fc9d45.png">

**Comment by anettrolikova** [comment]
use arbitrum 
<img width="530" alt="image" src="https://user-images.githubusercontent.com/44020788/218575213-f8e6ed1f-8ef1-48c5-a02c-31a37d25cc8c.png">

**Comment by aroralanuk** [comment]
Sounds good!

**Comment by nambrot** [comment]
@jmrossy Right now as scoped, we have folks transfer goerli ETH on the warp route UI and then swap on the uni UI. What would you estimate the scope of adding a page to our website that does the transfer and swap? I would suspect that it is pretty involved to get i to the same experience (like 3 days?), but wanted to check with you

**Comment by aroralanuk** [comment]
I'm using @uniswap-sdk for swapping arbiETH to GETH (our wrappedToken) and then call transferRemote here https://github.com/aroralanuk/faucetful-ui. We can handle the swap but for adding liquidity into the pool, we can link to the uniswap pool UI.

**Comment by jmrossy** [comment]
> @jmrossy What would you estimate the scope of adding a page to our website...

Yeah 3-5 days depending on the details and level of polish needed

**Comment by aroralanuk** [comment]
- [ ] Deployment of Goerli/Arbitrum ISMs on each other (and have them owned by the Abacus Works msig)
- [ ] Deployment of a custom IGP pair for this route that uses static gas oracle values for relaying to start
- [ ] Support HypNative in hyperlane-token
- [ ] Deployment of the warp route contracts (owned by Abacus Works msig)
- [ ] Deployment of the warp route UI
- [ ] Quick zap widget to swap ETH to GETH on arbitrum mainnet and bridge to ethereum goerli
- [ ] Write content around why this is cool, how hyperlane was uniquely positioned to do this (i.e. fully permissionless, best of both shared and isolated security), what it will mean for developers and the community, etc.
